cime_config/config_grids.xml

```diff
@@ -2777,6 +2825,34 @@
       <desc>SOwISC12to60E2r4 is a MPAS ice/ocean grid with enhanced resolution of 12km in the Southern Ocean around Antarctica. The high resolution regions smoothly transition to the background resolution of the standard low resolution 60to30km grid:</desc>
     </domain>
 
+    <domain name="FRISwISC08to60E3r1">
+      <nx>718054</nx>
+      <ny>1</ny>
+      <file grid="ice|ocn">$DIN_LOC_ROOT/share/domains/domain.ocn.FRISwISC08to60E3r1.230926.nc</file>
+      <desc>FRISwISC08to60E3r1 is a MPAS ice/ocean grid with enhanced resolution of 12km in the Southern Ocean around Antarctica and 4 km beneath the Filchner-Ronne Ice Shelf. The high resolution regions smoothly transition to the background resolution of the standard low resolution 60to30km grid:</desc>
+    </domain>
+
+    <domain name="FRISwISC04to60E3r1">
+      <nx>718054</nx>
```

Comment:
@irenavankova - the domain size should probably be different for each of the meshes

Reply:
Thank you, I fixed that and pushed the change.